assets/scripts/components/algolia/getHitData.js

```diff
@@ -1,25 +1,25 @@
 import { replaceSpecialCharacters } from "../../helpers/string";
+import { truncateContent, truncateContentAtHighlight } from "../../helpers/truncateContent";
 
 export function getHitData(hit, searchQuery = '') {
+    console.log(hit)
     const title = hit.title ? hit.title : hit.type;
     const cleanRelPermalink =
         hit.language == 'en' ? hit.relpermalink : hit.relpermalink.replace(`/${hit.language}/`, '');
 
     const matchingWordsArray = getFilteredMatchingWords(searchQuery).map(word => replaceSpecialCharacters(word))
     const joinedMatchingWordsFromSearch = matchingWordsArray.join('|');
     const regexQry = new RegExp(`(${joinedMatchingWordsFromSearch})`, 'gi');
-    const highlightedTitle = (hit._highlightResult.title.value || title);
-    const highlightedContent = (hit._highlightResult.content.value || '');
+    const highlightTitle = (hit._highlightResult.title.value || title);
+    const highlightContent = (hit._highlightResult.content.value || '');
 
     return {
         relpermalink: cleanRelPermalink,
         category: hit.category ? hit.category : 'Documentation',
         subcategory: hit.subcategory ? hit.subcategory : title,
-        title: handleHighlightingSearchResultContent(highlightedTitle, regexQry),
+        title: handleHighlightingSearchResultContent(highlightTitle, regexQry),
         section_header: hit.section_header || null,
-        content: handleHighlightingSearchResultContent(highlightedContent, regexQry),
-        content_snippet: hit._snippetResult ? hit._snippetResult.content.value : '',
-        content_snippet_match_level: hit._snippetResult ? hit._snippetResult.content.matchLevel : ''
```

Comment:
`content_snippet` and `content_snippet_match_level` were not referenced elsewhere in the code.